rcon/automods/seeding_rules.py

```diff
@@ -257,16 +259,40 @@ def punitions_to_apply(
                 if server_player_count >= dwc.max_players:
                     self._disable_for_round("disallowed_weapons")
 
+                ecf = self.config.enforce_cap_fight
+                if server_player_count >= ecf.max_players:
+                    self._disable_for_round("enforce_cap_fight")
+
                 violations = []
                 if (
-                    not self._is_seeding_rule_disabled("disallowed_roles")
-                    and drc.min_players <= server_player_count < drc.max_players
+                        not self._is_seeding_rule_disabled("disallowed_roles")
+                        and drc.min_players <= server_player_count < drc.max_players
                 ):
                     if aplayer.role in drc.roles:
                         violations.append(
                             drc.message.format(role=drc.roles.get(aplayer.role))
                         )
 
+                if not self._is_seeding_rule_disabled("enforce_cap_fight") and \
+                        (team == "axis" and game_state["axis_score"]) >= ecf.max_caps or \
+                        (team == "allies" and game_state["allied_score"] >= ecf.max_caps):
```

Comment:
Enforcing cap fight will cause problems on Offensive gamemodes. There should be a statement here to make sure this is only active on Warfare.

Reply:
Good point! 👍